exchange/exchange.go

```diff
@@ -27,11 +26,13 @@ import (
 	"github.com/prebid/prebid-server/metrics"
 	"github.com/prebid/prebid-server/openrtb_ext"
 	"github.com/prebid/prebid-server/prebid_cache_client"
+	"github.com/prebid/prebid-server/stored_requests"
 )
 
 type ContextKey string
 
 const DebugContextKey = ContextKey("debugInfo")
+const StoredRequestAttributes = "storedrequestattributes"
```

Comment:
I was also thinking about it too. And the thing is that `PrebidExtKey` constant is located in `openrtb_ext/device.go`. There are no bid.ext constants in bid.go. Not sure where this one should go.

Reply:
Yeah, I was wondering the same about `PrebidExtKey` but `PrebidExtKey` is a constant key that is valid for different extension objects such as bid and imp. So we want to have that in a common place. That common place definitely shouldn't be `device.go`.. maybe an `openrtb_ext.go` file that can hold common stuff like this but that's for another PR some other time :D. For now, I think `StoredRequestAttributes` would make more sense in `openrtb_ext/bid.go` since that is specific to bid ext.